Sources/TuistPluginSwiftLintFramework/Constants/Constants.swift

```diff
@@ -4,4 +4,13 @@ import Foundation
 enum Constants {
     /// The plugin version.
     static let version = "0.0.1"
+    
+    /// A set of constant values that reflects values from Tuist project.
+    enum Tuist {
+        /// A name of the directory that contains Tuist files.
+        static let tuistDirectoryName = "Tuist"
+        
+        /// A name of the directory that contains 3rd party dependencies installed using `Dependencies.swift`.
+        static let dependenciesDirectoryName = "Dependencies"
+    }
```

Comment:
Maybe it's worth to define the `Constants` in main tuist repo in a dedicated small module, and just depend on that from here?

Reply:
A target from the `ProjectAutomations` could have `isExternal` property.

Reply:
yes, I think it wouold be useful also to have it in TuistGraph.Target, so we don't need to rely on this kind of logic